### PR TITLE
Expose ServicePublishingStrategy option for kubevirt e2e

### PR DESCRIPTION
**Pull request by orenc1**
Currently the service publishing policy in the e2e suit is hard coded to the default Ingress, which utilizes usage of LoadBalancer. We would like to expose the option to use NodePort as well.

Signed-off-by: Oren Cohen <ocohen@redhat.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story*:
Fixes #

**Checklist**
- [ ] Subject and description added to both, commit and PR.
- [ ] Relevant issues have been referenced.
- [ ] This change includes docs. 
- [ ] This change includes unit tests.

### CODE REVIEW

File: test/e2e/e2e_test.go [hunk 1]
```diff
@@ -76,6 +76,7 @@ func TestMain(m *testing.M) {
 	flag.StringVar(&globalOpts.configurableClusterOptions.SSHKeyFile, "e2e.ssh-key-file", "", "Path to a ssh public key")
 	flag.StringVar(&globalOpts.platformRaw, "e2e.platform", string(hyperv1.AWSPlatform), "The platform to use for the tests")
 	flag.StringVar(&globalOpts.configurableClusterOptions.NetworkType, "network-type", "", "The network type to use. If unset, will default based on the OCP version.")
+	flag.StringVar(&globalOpts.configurableClusterOptions.ServicePublishingStrategy, "service-publishing-strategy", "Ingress", "Define how to expose the cluster services. Supported options: Ingress (Use LoadBalancer and Route to expose services), NodePort (Select a random node to expose service access through) (default \"Ingress\")")
```
Comment (davidvossel) [inline test/e2e/e2e_test.go:79]: can you call this command the `e2e.kubevirt-service-publishing-strategy`, and set the default value to the enum `kubevirt.IngressServicePublishingStrategy` 

My reasoning here is that the service publishing strategy appears to only be exposed as a KubeVirt option on the cli and not a platform wide option.

File: test/e2e/e2e_test.go [hunk 2]
```diff
@@ -234,6 +235,7 @@ type configurableClusterOptions struct {
 	NodePoolReplicas           int
 	SSHKeyFile                 string
 	NetworkType                string
+	ServicePublishingStrategy  string
```
Comment (davidvossel) [inline test/e2e/e2e_test.go:238]: Can you put this option up wiht the KubeVirt ones by giving it the `KubeVirt` prefix

Comment (orenc1) [reply test/e2e/e2e_test.go]: done.

File: test/e2e/e2e_test.go (hunk 1 above)
Comment (orenc1) [reply test/e2e/e2e_test.go]: done.

**Comment by orenc1** [comment]
currently this is not needed.
/close

**Pull request closed without merging**